crates/bevy_render/src/camera/bundle.rs

```diff
@@ -93,11 +93,16 @@ impl OrthographicCameraBundle {
     /// corresponding to `Z=+999.9` (closest to camera) to `Z=-0.1` (furthest away from
     /// camera) in world space.
     pub fn new_2d() -> Self {
+        Self::custom_2d(1000.0, 1.0)
+    }
+
+    /// Create an orthographic projection camera with a custom Z position and scale.
+    pub fn custom_2d(far: f32, scale: f32) -> Self {
```

Comment:
The other option for clarity here is that we could use custom `CameraPlane::Far(f32)` and `CameraScale(f32)` types for these.

A longer name is probably better, but I thought it was worth bringing up